orchestra/tests/test_task_lifecycle.py

```diff
@@ -498,7 +498,8 @@ def test_malformed_assignment_policy(self):
             slug='machine_step',
             is_human=False,
             assignment_policy={
-                'policy': 'previously_completed_steps',
+                'policy': ('orchestra.assignment_policies.'
```

Comment:
this way of describing functions is different from the way we describe machine functions.  we should keep them consistent, though i don't have an opinion which one changes